Tools/Plotfile/fvolumesum.cpp

```diff
@@ -120,7 +120,7 @@ void main_main()
                                                         problo[2]+static_cast<Real>(k+0.5)*dx[2])};
 
                                     // compute the volume
-                                    Real vol;
+                                    Real vol = NAN;
```

Comment:
How about using `std::numeric_limits<Real>::quiet_NaN()`?

Reply:
What is the advantage of doing so? I thought it will only cause a floating-point exception if it's used in an arithmetic operation, which it should never be here.

Reply:
In any case, I've changed it to quiet_NaN().

Reply:
`quiet_NaN()` is more explicit than `NAN` that it's a quiet NaN.  Also `NAN` is a float.  The compiler should be able to convert it to double at compile time, but it's probably not guaranteed.